resources/js/forms_editor/components/EditorHeader.vue

```diff
@@ -24,13 +24,33 @@
     </div>
     <div class="editor-header__actions">
       <a class="btn btn-link" :href="preview_url" target="_blank">プレビュー</a>
-      <button class="btn btn-primary" :disabled="is_saving">公開する</button>
+      <template v-if="is_public">
+        <span class="badge badge-primary mr-2">公開</span>
+        <button
+          class="btn btn-danger"
+          :disabled="is_saving"
+          v-on:click="togglePublic()"
```

Comment:
`v-on:` は `@` で書けるので、こちらにしましょう！
（Blade ファイルでは `v-on:` と書かないとダメですが、Vue ファイルでは OK です）

```suggestion
          @click="togglePublic()"
```